pages/report-repair/[route].js

```diff
@@ -226,6 +232,59 @@ function ReportRepair() {
           options = {[
             commonProblems.electricsLightsSwitches,
             commonProblems.wallsFloorAndCeiling,
+            { value: 'windows', title: 'Damaged or stuck windows'},
+          ]}
+        />
+      )
+    case 'repair-kitchen-window-problems':
+      return (
+        <RepairProblemBestDescription
+          handleChange={handleChange}
+          values={values}
+          options = {[
+            { value: 'smashed', title: 'Smashed window(s)'},
+            { value: 'stuckOpen', title: 'Window stuck open'},
+            { value: 'stuckShut', title: 'Window stuck shut'},
+            { value: 'condensation', title: 'Condensation'}
+          ]}
+        />
+      )
+    case 'repair-bathroom-window-problems':
+      return (
+        <RepairProblemBestDescription
+          handleChange={handleChange}
+          values={values}
+          options = {[
+            { value: 'smashed', title: 'Smashed window(s)'},
+            { value: 'stuckOpen', title: 'Window stuck open'},
+            { value: 'stuckShut', title: 'Window stuck shut'},
+            { value: 'condensation', title: 'Condensation'}
+          ]}
+        />
+      )
+    case 'repair-bedroom-window-problems':
+      return (
+        <RepairProblemBestDescription
+          handleChange={handleChange}
+          values={values}
+          options = {[
+            { value: 'smashed', title: 'Smashed window(s)'},
+            { value: 'stuckOpen', title: 'Window stuck open'},
+            { value: 'stuckShut', title: 'Window stuck shut'},
+            { value: 'condensation', title: 'Condensation'}
+          ]}
+        />
+      )
+    case 'repair-living-areas-window-problems':
```

Comment:
rather than having this duplicated a bunch of times, can't we have a generic `window-problems` page, similar to how we have `wall-floor-ceiling-problems`

Reply:
ohh yes, good idea